Cargo.toml

```diff
@@ -21,7 +21,7 @@ members = [
 
 [package]
 name = "skywalking-php"
-version = "0.7.0"
+version = "0.8.0-dev"
 authors = ["Apache Software Foundation", "jmjoy <jmjoy@apache.org>", "Yanlong He <heyanlong@apache.org>"]
 description = "Apache SkyWalking PHP Agent."
 edition = "2021"
```

Comment:
Do we need to update this?

Reply:
Yes, we need to start a new version development.